src/core/server/saved_objects/saved_objects_service.test.ts

```diff
@@ -169,6 +170,27 @@ describe('SavedObjectsService', () => {
         expect(typeRegistryInstanceMock.registerType).toHaveBeenCalledWith(type);
       });
     });
+
+    describe('#setRepositoryFactoryProvider', () => {
```

Comment:
I saw cypress test failed, I'm assuming that the failed cypress test should not related to your PR. maybe double check before merge in. Thanks

Reply:
The failed cypress test is not related to this PR.